include/avif/avif.h

```diff
@@ -994,6 +1102,11 @@ typedef struct avifDecoder
     // avifDecoderNextImage() or avifDecoderNthImage(), as decoder->image->alphaPlane won't exist yet.
     avifBool alphaPresent;
 
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+    // This is true when avifDecoderParse() detects a gain map.
+    avifBool gainMapPresent;
+#endif
+
```

Comment:
In anticipation of the libavif 1.0.0 release, the new member `gainMapPresent` should be added to the end of the `avifDecoder` struct, after all the current struct members.